care/facility/api/serializers/patient.py

```diff
@@ -503,6 +503,20 @@ class Meta:
         exclude = ("patient_note",)
 
 
+class ReplyToPatientNoteSerializer(serializers.ModelSerializer):
+    created_by_object = UserBaseMinimumSerializer(source="created_by", read_only=True)
```

Comment:
```suggestion
    id = serializers.CharField(source="external_id", read_only=True)
    created_by_object = UserBaseMinimumSerializer(source="created_by", read_only=True)
```

the `id` in field would return the db `id`, we should avoid exposing it